Presentation.Web/Controllers/API/V2/External/ItSystemUsages/ItSystemUsageV2Controller.cs

```diff
@@ -436,6 +437,62 @@ public IHttpActionResult DeleteSystemUsageRelation([NonEmptyGuid] Guid systemUsa
                 .Match(FromOperationError, () => StatusCode(HttpStatusCode.NoContent));
         }
 
+        [HttpPost]
+        [Route("{systemUsageUuid}/external-references")]
+        [SwaggerResponse(HttpStatusCode.Created)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        public IHttpActionResult AddExternalReference([NonEmptyGuid] Guid systemUsageUuid, [FromBody] ExternalReferenceDataWriteRequestDTO dto)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            var properties = _writeModelMapper.MapExternalReference(dto);
+            
+            return _writeService
+                .AddExternalReference(systemUsageUuid, properties)
+                .Select(_responseMapper.MapExternalReferenceDTO)
+                .Match(reference => Created($"{Request.RequestUri.AbsoluteUri.TrimEnd('/')}/{systemUsageUuid}/external-references/{reference.Uuid}", reference), FromOperationError);
+        }
+
+
+
+        [HttpPut]
+        [Route("{systemUsageUuid}/external-references/{externalReferenceUuid}")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
```

Comment:
missing "badrequest" response code and it's actually the first possible response code if you step into the function